server.js

```diff
@@ -20,4 +23,64 @@ app.get("/", function (request, response) {
   response.sendFile(__dirname + "/index.html");
 });
 
+//Create a new message
+app.post("/messages", (req, res) => {
+  let newPost = {
```

Comment:
I'd recommend using `const` here when initiating your newPost object. A good habit to get into is to use const everywhere until you have to use let, that way you're ensuring your variables are as protected as possible.